apps/web/src/common/context/UserContext/hooks/useUserData/index.ts

```diff
@@ -0,0 +1,29 @@
+import { UserType } from 'common/types/prisma'
+import React, { useEffect } from 'react'
+
+import { UserData } from '../../types'
+
+const useUserData = () => {
+  const [user, setUser] = React.useState<UserData | null>(null)
+  const [isLoading, setLoading] = React.useState(true)
+
+  useEffect(() => {
+    const getUser = async () => {
+      // TODO fetch user data from API
+      // will write test for this file after API call is implemented
+      setTimeout(() => {
```

Comment:
(Optional)
Please consider making cleanup function for setTimeout

Reply:
let's ignore it since it will be changed to API call soon